telegram/ext/choseninlineresulthandler.py

```diff
@@ -70,6 +74,12 @@ class ChosenInlineResultHandler(Handler[Update]):
             DEPRECATED: Please switch to context based callbacks.
         run_async (:obj:`bool`): Determines whether the callback will run asynchronously.
             Defaults to :obj:`False`.
+        pattern (:obj:`str` | `Pattern`, optional): Regex pattern. If not :obj:`None`, ``re.match``
+            is used on :attr:`telegram.ChosenInlineResult.result_id` to determine if an update
+            should be handled by this handler. This is accessible in the callback as
+            `context.matches`.
```

Comment:
```suggestion
            ``context.matches``.
```
or even

```suggestion
            :attr:`telegram.ext.CallbackContext.matches`.
```
?

Reply:
I thought about the actual link, but for some reason I thought I want to use our naming convention. Having the link generated there is way better though.